packages/connect-webextension/package.json

```diff
@@ -22,7 +22,7 @@
     ],
     "main": "src/index.ts",
     "publishConfig": {
-        "main": "lib/index.js"
+        "main": "build/trezor-connect-webextension.js"
```

Comment:
how about the minified version? does using unminified help with debugging?

Reply:
I will check. The minified version is actually not that minified - no renaming variables or formatting, probably it just has pruned unused code. It might make sense to only ship that

Reply:
Seems that the minified version doesn't actually work. Not even with importScripts

Reply:
I would probably just remove the minified version